lib/BridgeAbstract.php

```diff
@@ -8,13 +8,22 @@
 	const MAINTAINER = 'No maintainer';
 	const CACHE_TIMEOUT = 3600;
 	const PARAMETERS = array();
-
+	
```

Comment:
Not sure if this tab will break Travis (we will find out on the next commit :smile:), but white-space should be avoided here.